pkg/utils/common.go

```diff
@@ -33,8 +33,9 @@ import (
 // move to tidb-tools later
 
 const (
-	maxRetryCount = 3
-	retryTimeout  = 3 * time.Second
+	maxRetryCount  = 3
+	retryTimeout   = 3 * time.Second
+	stringLenLimit = 1024
```

Comment:
can we only define it in untils/string.go, and use `-1` as argument in `TruncateString` if use default stringLenLimit?

Reply:
done in https://github.com/pingcap/dm/pull/257/commits/02aa7c3d02ce604be1d3f8ec03e59aefbe60d7c2.